src/app/garfile/amg/checkin/pane.njk

```diff
@@ -28,27 +29,54 @@
   <tr class="govuk-table__row">
     <td class="govuk-table__cell">
       <div>
-        <div class="status-disc {{statuscolour}}"></div>
+        <div class="status-disc {{statuscolour}}" title="{{statustitle}}"></div>
       </div>
     </td>
     <td class="govuk-table__cell">{{person.firstName}}
       {{person.lastName}}</td>
-    <td class="govuk-table__cell">
+    <td class="govuk-table__cell" style="position:relative;">
```

Comment:
Nit (optional): Awkward hover effect (does lead to more contrast issues).

Reply:
<img width="1440" alt="Screenshot 2023-11-22 at 10 48 01" src="https://github.com/UKHomeOffice/egar-public-site-ui/assets/123387957/993c7a56-1766-4e1e-9a13-930d4045bf39">